examples/evm/coverage.py

```diff
@@ -3,7 +3,7 @@
 m = ManticoreEVM()
 m.verbosity(3)
 #And now make the contract account to analyze
-source_code = file('coverage.sol').read()
+source_code = open('coverage.sol').read()
```

Comment:
Should we use:
```python
with open('coverage.sol') as in_file:
    source_code = in_file.read()
```
to ensure that the file is closed and the file handle is garbage collected on all Python implementations?